ttnn/cpp/pybind11/device.cpp

```diff
@@ -104,6 +104,25 @@ void py_device_module_types(py::module& m_device) {
     py::class_<SubDeviceId>(m_device, "SubDeviceId", "ID of a sub-device.");
 
     py::class_<SubDeviceManagerId>(m_device, "SubDeviceManagerId", "ID of a sub-device manager.");
+
+    py::class_<tt::tt_metal::detail::MemoryView>(
+        m_device, "MemoryView", "Class representing view of the memory (dram or l1)")
+        .def_readonly("num_banks", &tt::tt_metal::detail::MemoryView::num_banks)
+        .def_readonly(
+            "total_allocatable_per_bank_size_bytes",
```

Comment:
what exactly does this mean? amount of memory per bank? or amount of free memory per bank?
why "total" but at the same time "per bank"?